.changes/next-release/Bug Fix-97552318-ef7f-443c-888f-bc0102342386.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Show correct prompts on iteration limits for featureDev"
```

Comment:
Can you adjust the text here to be more in-line with the other changelog items e.g. starting it with: `Amazon Q Feature Dev`. Maybe something like: `Amazon Q Feature Dev: Fix followups after hitting iteration limit`